app/views/shared/_landing_strip.html.haml

```diff
@@ -2,4 +2,5 @@
   = t('landing_strip_html',
     name: content_tag(:span, display_name(account), class: :emojify),
     domain: site_hostname,
+    root_path: root_path,
```

Comment:
Since this value: `<a href="%{root_path}"><strong>%{domain}</strong></a>` is the exact same in every translation, would it be possible to prepare this from the partial and send it a value that's already an html link (and do the right thing re: html_safe) so that the partial is something like:

    link_to_root_path: link_to(...)

and then the i18n like:

    landing_strip_html: '.... %{link_to_root_path} ...'

?